sass/layouts/_header.scss

```
@@ -0,0 +1,380 @@
+.header {
+    &__top {
+        background-color: $third-color;
+        border-bottom: 1px solid $border-color;
+    }
+    &__top-row {
+        display: flex;
+        justify-content: space-between;
+    }
+    &__top-list {
+        display: flex;
+        margin: 0;
+    }
+    &__top-item:first-child {
```

```suggestion
    }
    
    &__top-row {
        display: flex;
        justify-content: space-between;
    }
    
    &__top-list {
        display: flex;
        margin: 0;
    }
    
    &__top-item:first-child {
```
Mỗi block cách nhau 1 dòng